### PR TITLE
Create FCM's CHANGELOG.md with a release note.

### PR DESCRIPTION
**Pull request by zwu52**
FCM starts to have CHANGELOG.md for release notes (Thanks to the JS core and DevRel folks for creating/reviewing the previous release notes).

The first entry include the release note  that is meant for the release scheduled on 2020-04-02.
The fix will address the issues in https://github.com/firebase/firebase-js-sdk/issues/2712

### CODE REVIEW

File: packages/messaging/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Unreleased
+- [fixed] Fixed an issue introduced in 7.7.0 where the FCM switched to provide base64-encoded VAPID key
```
Comment (egilmorez) [inline packages/messaging/CHANGELOG.md]: "Fixed an issue where" is a phrase often followed by what the issue is.  

I think you may mean "an issue introduced in 7.7.0, when FCM switched to providing base64-encoded VAPID keys..."

File: packages/messaging/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+to [PushManager](https://developer.mozilla.org/en-US/docs/Web/API/PushManager) for push 
+subscribtion. For backward compatibility, the SDK switched back to use VAPID key in type (ArrayBuffer)[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer].
```
Comment (egilmorez) [inline packages/messaging/CHANGELOG.md]: Suggest "the SDK has switched back to using VAPID keys..."

Comment (egilmorez) [inline packages/messaging/CHANGELOG.md]: Looks like this markdown link swaps the placement of parens and square brackets.

File: packages/messaging/CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Unreleased
+- [fixed] Fixed an issue introduced in 7.7.0, when FCM switched to provide base64-encoded VAPID 
+key to [PushManager](https://developer.mozilla.org/en-US/docs/Web/API/PushManager) for push subscribtion. For backward compatibility, the SDK has switched back to using VAPID key in type [ArrayBuffer](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer).
```
Comment (hsubox76) [inline packages/messaging/CHANGELOG.md]: subscribtion -> subscription

Comment (zwu52) [reply packages/messaging/CHANGELOG.md]: good catch.

**Pull request merged**